webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux/f1';
+
+// import from a file named `foo.json` located in the current directory:
+import f2 from './foo/f2';
```

Comment:
Need to include the file extension here! Webpack will resolve it if it was a `.js`, `.jsx`, `.ts`, `.tsx`, but since it's none of those, it won't know where to import from